### PR TITLE
enhancement: add sample example in the commands

### PR DESCRIPTION
**Pull request by shay1760**
**Description of your changes:**
adding small enhancement, adding examples for required commands, using small caps for error logging.

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)

### CODE REVIEW

File: cmd/commands/operator.go [hunk 1]
```diff
@@ -42,9 +42,10 @@ var restartCmd = &cobra.Command{
 }
 
 var setCmd = &cobra.Command{
-	Use:   "set",
-	Short: "Set the property in the rook-ceph-operator-config configmap.",
-	Args:  cobra.ExactArgs(2),
+	Use:     "set",
+	Short:   "Set the property in the rook-ceph-operator-config configmap.",
+	Example: "kubectl rook-ceph operator set <KEY> <VALURE>",
```
Comment (subhamkrai) [inline cmd/commands/operator.go:47]:
```suggestion
	Example: "kubectl rook-ceph operator set <KEY> <VALUE>",
```

**Review by subhamkrai — CHANGES_REQUESTED** [review]

File: cmd/commands/operator.go (hunk 1 above)
Comment (shay1760) [reply cmd/commands/operator.go]: updated

File: cmd/commands/restore.go [hunk 2]
```diff
@@ -23,9 +23,10 @@ import (
 
 // RestoreCmd represents the restore commands
 var RestoreCmd = &cobra.Command{
-	Use:   "restore-deleted",
-	Short: "Restores a CR that was accidentally deleted and is still in terminating state. Ex: restore cephcluster <my-cluster>",
-	Args:  cobra.MinimumNArgs(1),
+	Use:     "restore-deleted",
+	Short:   "Restores a CR that was accidentally deleted and is still in terminating state. Ex: restore cephcluster <my-cluster>",
```
Comment (subhamkrai) [inline cmd/commands/restore.go:27]:
```suggestion
	Short:   "Restores a CR that was accidentally deleted and is still in terminating state.,
```

remove this as not needed anymore

Comment (shay1760) [reply cmd/commands/restore.go]: updated.

**Review by subhamkrai — APPROVED** [review]

**Comment by subhamkrai** [comment]
Thanks for the pr @shay1760

**Pull request merged**